wicket-jmx/src/main/java/org/apache/wicket/jmx/DebugSettingsMBean.java

```diff
@@ -71,6 +71,8 @@ public interface DebugSettingsMBean
 	 * Enables wrapping output of markup container in html comments that contain markup container's
 	 * class name. (Useful for determining which part of page belongs to which markup file).
 	 * 
+	 * //TODO WICKET-6890 JMX Probably does not work with enum, What do we do here?
```

Comment:
@martin-g: Any idea what we should do about the JMX bean? Looking at all available beans, enums have been carefully avoided.

Reply:
One way is to use the enum name as a String.
Another way is to use MXBean instead of MBean.

Reply:
I went the "enum name as a String" way.